pkgs/applications/graphics/pinta/default.nix

```
@@ -37,23 +41,25 @@ buildDotnetPackage rec {
       "Mono\\.Addins\\.Setup"
     ];
 
-    stripVersion = name: file: let
-        match = ''<Reference Include="${name}([ ,][^"]*)?"'';
-        replace = ''<Reference Include="${name}"'';
-      in "sed -i -re 's/${match}/${replace}/g' ${file}\n";
+    stripVersion = version: file:
```

```suggestion
    stripVersion = name: file:
```

that is a function variables unrelated to pname+version or name.